records_mover/db/bigquery/bigquery_db_driver.py

```diff
@@ -135,6 +134,6 @@ def tweak_records_schema_after_unload(self,
             # DATETIME field."
             #
             # BigQuery exports this as an Avro string type
-            return records_schema.convert_datetimes_to_string()
+            return records_schema.cast_field_types({'datetime': 'string'})
```

Comment:
Since we seem to be using it more and more, let's at least generalize this function a bit.